src/test/java/com/karankumar/bookproject/backend/repository/BookRepositoryTest.java

```diff
@@ -73,11 +80,7 @@ void successfullyDeleteABook_whenAuthorHasOtherBooks() {
         bookRepository.delete(book);
 
         // then
-        assertSoftly(softly -> {
-            softly.assertThat(bookRepository.findAll().size()).isOne();
-            softly.assertThat(bookRepository.findAll().get(0))
-                  .isEqualToComparingFieldByField(book2);
-        });
+        assertThat(bookRepository.findAll().size()).isEqualTo(3);
```

Comment:
To improve readability, could you please create a local `Author` that has two books, delete one of the books and then assert that one book remains